src/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs

```diff
@@ -328,6 +328,22 @@ public void Match_Timeout()
             Assert.Equal("a", match.Value);
         }
 
+        [Fact]
+        public void Match_Timeout_Throws()
+        {
+            RemoteInvoke(() =>
+            {
+                const string DefaultMatchTimeout_ConfigKeyName = "REGEX_DEFAULT_MATCH_TIMEOUT";
+                const string Pattern = @"^([0-9a-zA-Z]([-.\w]*[0-9a-zA-Z])*@(([0-9a-zA-Z])+([-\w]*[0-9a-zA-Z])*\.)+[a-zA-Z]{2,9})$"; // 723 msec
```

Comment:
does this mean it normally takes 723ms on your box? can we do something more pathologically slow instead? we've seen just about every possible timing issue on tests over time including timeouts that dont fire until long after they were expected to.

Reply:
something that would take 30 seconds or whatever perhaps

Reply:
just for reference, this test is a "backport" from netfx.